next.config.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @type {import('next').NextConfig}
+ */
+const nextConfig = {
```

Comment:
Do we also need to configure `basepath` without trailing slash?

Looking at here for reference: https://github.com/actions/configure-pages/blob/main/src/set-pages-config.js#L26-L43

Reply:
No we do not need to configure basepath

Reply:
we do not even need image optimisation in this file since `static_site_generator: next` is taking care of that.